src/popup/components/SwapRates.vue

```diff
@@ -62,13 +63,14 @@ export default defineComponent({
   },
   setup(props) {
     const { getProtocolAvailableTokens } = useFungibleTokens();
+    const { tokenSwapAddressToTokenContractAddress } = useAeTokenSwaps();
 
-    const { isDexSwap, txFunctionParsed } = useTransactionData({
+    const { isDexSwap, txFunctionParsed, isTokenSwap } = useTransactionData({
       transaction: toRef(() => props.transaction),
     });
 
     const rates = computed(() => {
-      if (!isDexSwap.value || !txFunctionParsed.value) {
+      if (!(isDexSwap.value || isTokenSwap.value) || !txFunctionParsed.value) {
```

Comment:
Isn't this equal to:
```suggestion
      if (!isDexSwap.value || !isTokenSwap.value || !txFunctionParsed.value) {
```